Common/FeatureFlags.swift

```diff
@@ -75,10 +75,11 @@ struct FeatureFlagConfiguration: Decodable {
         self.fiaspInsulinModelEnabled = true
         #endif
         
-        #if OBSERVE_HEALTHKIT_FOR_CURRENT_APP_ONLY
-        self.observeHealthKitForCurrentAppOnly = true
+        // Swift compiler config is inverse, since the default state is enabled.
```

Comment:
Shouldn't there also be a change to `Tidepool/Loop/LoopOverride.xcconfig` to add `OBSERVE_HEALTH_KIT_SAMPLES_FROM_OTHER_APPS_DISABLED`?

Reply:
That file is in LoopWorkspace (which I did not yet submit for review).

Reply:
Gotcha